frontend/tests/ui/util.py

```diff
@@ -139,12 +139,26 @@ def assert_no_js_errors(driver, *, path_whitelist=(), message_whitelist=()):
                 continue
             if is_path_whitelisted(entry['message'], path_whitelist):
                 continue
-            if is_message_whitelisted(entry['message'], message_whitelist):
+            if message_matches(entry['message'], message_list):
                 continue
             unexpected_errors.append(entry['message'])
         assert not unexpected_errors, '\n'.join(unexpected_errors)
 
 
+@contextlib.contextmanager
+def assert_js_errors(driver, *, message=()):
+    '''Shows in a list unexpected errors in javascript console'''
```

Comment:
```suggestion
    '''Asserts that a JavaScript error is logged in the console'''
```

aquí no son errores inesperados: al contrario, _esperas_ que ocurran.

Reply:
Listo